pose-detection/src/blazepose/blazepose_test.ts

```diff
@@ -135,3 +133,65 @@ describeWithFlags('Blazepose static image ', BROWSER_ENVS, () => {
     });
   });
 });
+
+describeWithFlags('Blazepose video ', BROWSER_ENVS, () => {
+  let detector: poseDetection.PoseDetector;
+  let timeout: number;
+  let expectedFull: number[][][];
+
+  beforeAll(async () => {
+    timeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;  // 2mins
+
+    expectedFull = await fetch(`${KARMA_SERVER}/pose_squats.full_body.json`)
+                       .then(response => response.json());
+    expectedFull = expectedFull.map(frameResult => {
+      return frameResult.map(keypoint => [keypoint[0], keypoint[1]]);
+    });
+  });
+
+  afterAll(() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = timeout;
+  });
+
+  UPPERBODY_ONLY.forEach(upperBodyOnly => {
+    it('test.', async () => {
+      // Note: this makes a network request for model assets.
+      const modelConfig:
+          poseDetection.BlazeposeModelConfig = {quantBytes: 4, upperBodyOnly};
+      detector = await poseDetection.createDetector(
+          poseDetection.SupportedModels.MediapipeBlazepose, modelConfig);
+
+      const result: number[][][] = [];
+
+      const callback = async(video: HTMLVideoElement): Promise<void> => {
+        const poses = await detector.estimatePoses(video);
+        result.push(poses[0].keypoints.map(kp => [kp.x, kp.y]));
+      };
+      await loadVideo('pose_squats.mp4', callback);
+
+      let count = 0;
+      for (let i = 0; i < result.length; i++) {
+        const frameResult = result[i];
+        for (let j = 0; j < frameResult.length; j++) {
+          const keypoint = frameResult[j];
+          const x = keypoint[0];
+          const y = keypoint[1];
+
+          const ex = expectedFull[i][j][0];
+          const ey = expectedFull[i][j][1];
+
+          if (Math.abs(x - ex) > 30 || Math.abs(y - ey) > 30) {
+            count++;
+            console.log(`Frame ${i}, Keypoint ${j}: Actual [${x}, ${
+                y}], Expected [${ex}, ${ey}]`);
+          }
+        }
+      }
+      console.log(count);
+      // expectArraysClose(result.slice(0, 65), expectedFull, 45);
```

Comment:
Please uncomment and reuse the constant introduced above.
And increase the constant to 45 (or some smaller value if possible).